lua/exporters/object.lua

```diff
@@ -45,7 +45,8 @@ GRPC.exporters.unit = function(unit)
     groupName = Unit.getGroup(unit):getName(),
     numberInGroup = unit:getNumber(),
     heading = heading,
-    speed = speed
+    speed = speed,
+    category = unit:getGroup():getCategory(),

```

Comment:
There is also a `unit:getCategory()` but that seems to be deprecated as it always returns `0`